test/test.js

```diff
@@ -292,4 +292,21 @@ describe('BigNumber.js', function () {
             BigNumber(5).plus(97).minus(53).plus(434).multiply(5435423).add(321453).multiply(21).div(2).val().should.equal("27569123001");
         });
     });
+
+    describe('very big numbers', function () {
+        it('should test main arithmetic on very large integers', function () {
```

Comment:
There are already tests defined for each API method. Please move these ones there.